### ISSUE TITLE
Model weights for CT

### ISSUE THREAD

**Issue by MehriB** [Hendrik-code/spineps#60]
Hello, 

Great work. 
In the paper it is mentioned both CT and MRI. Are the model weights also available for CT?

Thank you,
Mehri

**Comment by MehriB** [comment]
Hi @Hendrik-code . Thank you for the great work and sharing.
Is it possible to get the code on full VERIDAH training? I would like to train the labeling model for my CT data? Thank you.

Best regards,
Mehri

**Comment by Hendrik-code** [comment]
Hey @MehriB 

The labeling model is available in CT, but the segmentation is not. But we are working on it and actually should have a release for a CT segmentation soon.

Reg. VERIDAH training code: The released model weights of the CT labeling don't work well on your data? Please confirm this, then I can make the training code readable and send it to via DM.

**Comment by MehriB** [comment]
Hi @Hendrik-code. Thank you for the response.

Looking forward to try your CT segmentation model once is out :)

On my side I pre-generated the CT segmentation and tried the labeling step using -ml labeling (I assume that is the right one for CT, among these three options that I got:   options are ['t2w_labeling', 'labeling', 't2w_labeling_v1.4.0'])

and the result was not what I was expecting, see attached. 
I have to mention that in my segmentation for the spinus and transverse processes, I used one mask and didn't have different sub parts as you have for MRI.

It would be great if you could share the training code of VERIDAH. Here is my email: mehri.baniasadi92@gmail.com

Thank you 

<img width="2025" height="577" alt="Image" src="https://github.com/user-attachments/assets/ad0c06ce-3819-4c12-84fa-36e588bdcab2" />

**Comment by robert-graf** [comment]
The first stage is in testing on branch "loading" #61 and TPTBox branch "development_robert". 

Using the older second stage cases issues for C1 and merged vertebra, but for the rest it works. 

We work on training a second stage.

**Comment by MehriB** [comment]
@robert-graf Thanks for the update. I will try the new model :)
If I understood correctly, 
The first step of instance segmentation is what is added (-ms ct)
The second you are currently working on  (-mv )
Labeling, I have the checkpoint. (-ml ct_labeling)

**Comment by robert-graf** [comment]
I did not know that we had a ct_labeling just existing here. And it was never linked properly in the repo.

In the testing branch "loading" you will be able to automatically download it under`-ml  ct_labeling.`

Usually, I would test it more, but it worked on the first 4 samples. So, no obvious regression so far.  

`-ms ct` is the new semantic model.

-mv needs to be redone, as there are things we never encounter in MRI segmentations like the C1 and merged bones, but it works ok enough.

**Comment by robert-graf** [comment]
We have now added the second phase for CT.
